src/media/css/elements--header.styl

```diff
@@ -145,10 +156,13 @@ mkt-header-child {
     display: inline;
     line-height: 40px;
     margin: 0;
+    max-width: 100%;
     opacity: .75;
-    padding: 15px 0;
+    overflow: hidden;
+    padding: 15px 40px;
     position: absolute;
     top: 50%;
+    transition: 250ms;
```

Comment:
I made this transition when typing starts at `100ms`. Left the fade out at `250ms`. Should they both be `100ms`?